components/server/ee/src/prebuilds/start-prebuild-context-parser.ts

```diff
@@ -19,14 +19,8 @@ export class StartPrebuildContextParser implements IPrefixContextParser {
     }
 
     public async handle(user: User, prefix: string, context: WorkspaceContext): Promise<WorkspaceContext> {
-        if (IssueContext.is(context)) {
-            throw new Error("cannot start prebuilds on an issue context");
-        }
-
-        const result: StartPrebuildContext = {
-            title: `Prebuild of "${context.title}"`,
-            actual: context,
-        };
-        return result;
+        throw new Error(
```

Comment:
Simlarly here, is it possible to throw an InvalidArgument error?

Reply:
In this case, it's more a case of a no-longer-valid route (`gitpod.io/#prebuild/` URL prefix is deprecated), so it's not really an invalid argument. 🤔 Also, I'm not exactly sure a specific error would bubble correctly all the way to the user.

Let's see, I'll give this a quick try, but leave as is if it turns out too complicated for now.

Reply:
Given this is the API, and you supplied something which is not valid anymore, I think it would be an Invalid Argument. You could also go with Not Found, but I think that doens't communicate the fact that the client to this API specified something which is no longer valid.

Reply:
Interesting. I guess this could also be a `400 Bad Request` or a `410 Gone`?